scene/resources/fog_material.cpp

```diff
@@ -159,7 +159,7 @@ uniform sampler3D density_texture: hint_default_white;
 void fog() {
     DENSITY = density * clamp(exp2(-height_falloff * (WORLD_POSITION.y - OBJECT_POSITION.y)), 0.0, 1.0);
     DENSITY *= texture(density_texture, UVW).r;
-    DENSITY *= pow(clamp(-SDF / min(min(EXTENTS.x, EXTENTS.y), EXTENTS.z), 0.0, 1.0), edge_fade);
+    DENSITY *= pow(clamp(-SDF / min(min(SIZE.x, SIZE.y), SIZE.z) / 2, 0.0, 1.0), edge_fade);
```

Comment:
```suggestion
    DENSITY *= pow(clamp(-2.0 * SDF / min(min(SIZE.x, SIZE.y), SIZE.z), 0.0, 1.0), edge_fade);
```
You can't divide a float by an integer in GLSL, also a division by a division is a multiplication, so switching this to use multiplication by 2.0 should solve the bug and make it slightly faster

Reply:
Sorry, I don't quite get that.
How is `-SDF / X / 2.0` the same as `-2.0 * SDF / X`?

Reply:
It works if you imagine parentheses around ``min(min(SIZE.x, SIZE.y), SIZE.z) / 2,`` like I somehow did haha!

You are right, it should be ``-0.5 * SDF / min(min(SIZE.x, SIZE.y), SIZE.z)``

Reply:
Ahh, yes, makes sense. Will change it. :)